src/components/node/banner_alert/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { isEmpty } from 'lodash'
+import { useRef, useEffect, useState } from 'react'
+import { recordEvent } from '@/utils/recordEvent'
+import { regionBaseURL } from '@/utils/helpers'
+import { VaBanner } from '@department-of-veterans-affairs/component-library/dist/react-bindings'
+
+const BannerAlert = ({ node }): JSX.Element => {
+  const [isClicked, setIsClicked] = useState(false)
+  const analyticsRef = useRef(null)
+
+  useEffect(() => {
+    function handler(event) {
+      if (analyticsRef.current?.contains(event.target)) {
+        setIsClicked(true)
+      }
+    }
+    window.addEventListener('click', handler)
+    return () => window.removeEventListener('click', handler)
+  }, [])
+
+  if (isEmpty(node)) return
+
+  const hideOnSubpages = node.field_alert_inheritance_subpages
+  const alertType =
+    node.field_alert_type === 'information' ? 'info' : node.field_alert_type
+  const region = '/' + regionBaseURL(node.path.alias)
+  const lastArg = node.path.alias.substring(node.path.alias.lastIndexOf('/'))
+
+  let body = node.field_body.processed
+  let outputStatus = false
+  let statusUrl = ''
+
+  node.field_banner_alert_vamcs?.map((vamcs) => {
+    if (region == vamcs.field_office.path.alias) {
+      outputStatus = true
+    }
+    if (hideOnSubpages && lastArg != region && lastArg != '/operating-status') {
+      outputStatus = false
+    }
+    statusUrl = vamcs.path.alias
+  })
+
+  if (node.field_alert_operating_status_cta && statusUrl.length) {
+    body += `<p>
+          <a href='${statusUrl}'>
+            Get updates on affected services and facilities
+          </a>
+      </p>`
+  }
+
+  if (node.field_alert_find_facilities_cta) {
+    body += `
+      <p>
+        <a href="/find-locations">Find other VA facilities near you</a>
+      </p>`
+  }
+
+  if (isClicked) {
+    let eventData = {}
+
+    if (node.field_alert_operating_status_cta && statusUrl.length) {
+      eventData = {
+        event: 'nav-warning-alert-box-content-link-click',
+        alertBoxHeading: `${node.title}`,
+      }
+    } else {
+      eventData = {
+        event: 'nav-alert-box-link-click',
+        'alert-box-status': alertType,
+        'alert-box-headline': node.title,
+        'alert-box-headline-level': '3',
+        'alert-box-background-only': 'false',
+        'alert-box-closeable': 'false',
+        'alert-box-click-label': '$2',
+      }
+    }
+    recordEvent(eventData)
+    setIsClicked(false)
+  }
+
+  return (
+    <VaBanner
+      id={node.id}
+      role="va-banner"
+      showClose={node.field_alert_dismissable}
+      headline={node.title}
+      type={node.field_alert_type}
+      visible={outputStatus}
+      windowSession={node.field_alert_dismissable == 'dismiss-session'}
+    >
+      <div ref={analyticsRef} dangerouslySetInnerHTML={{ __html: body }} />
+    </VaBanner>
+  )
+}
```

Comment:
We need the `meta` object exported somewhere here. Even though we're not using banner_alert in a more conventional page context, we rely on the node meta for other purposes, including letting `<Node />` know what component will render objects of type `node--full_width_banner_alert`.

It can be modeled after News Story: https://github.com/department-of-veterans-affairs/next-build/blob/main/src/components/node/news_story/index.tsx#L148

We won't need `params` for this; that should be able to be null.

Reply:
Will do